api_spec.yaml

```diff
@@ -1266,17 +1472,285 @@ paths:
                                                     properties:
                                                         enabled:
                                                             type: boolean
-
                                                 thirdParty:
                                                     type: object
                                                     properties:
                                                         enabled:
                                                             type: boolean
+    /dashboard/api/tenants/user/associate:
```

Comment:
remove this API, no need to implement it now.

Reply:
Done